mod/profile_photo.php

```diff
@@ -125,8 +125,8 @@ function profile_photo_post(App $a)
 
 				info(L10n::t('Shift-reload the page or clear browser cache if the new photo does not display immediately.') . EOL);
 				// Update global directory in background
-				if ($url && strlen(Config::get('system', 'directory'))) {
-					Worker::add(PRIORITY_LOW, "Directory", $url);
+				if ($path && strlen(Config::get('system', 'directory'))) {
+					Worker::add(PRIORITY_LOW, "Directory", $path);
```

Comment:
Can this Worker work with a relative path instead of a fully url?

Because otherwise I have to change the goaway to `System::externalRedirect()` with `$a->getBaseURL()` which is now bad practice

Reply:
It is bad practice if it can be avoided, but you can make a filter and add a `@TODO` to refactor the behavior later.